gcp_variant_transforms/libs/bigquery_vcf_schema.py

```diff
@@ -134,16 +139,13 @@ def generate_schema_from_header_fields(
       continue
     schema.fields.append(bigquery.TableFieldSchema(
         name=bigquery_util.get_bigquery_sanitized_field_name(key),
-        type=bigquery_util.get_bigquery_type_from_vcf_type(field.type),
-        mode=_get_bigquery_mode_from_vcf_num(field.num),
-        description=bigquery_util.get_bigquery_sanitized_field(field.desc)))
+        type=bigquery_util.get_bigquery_type_from_vcf_type(
+            field[_HeaderKeyConstants.TYPE]),
+        mode=bigquery_util.get_bigquery_mode_from_vcf_num(
+            field[_HeaderKeyConstants.NUM]),
+        description=bigquery_util.get_bigquery_sanitized_field(
+            field[_HeaderKeyConstants.DESC])))
     info_keys.add(key)
   if variant_merger:
     variant_merger.modify_bigquery_schema(schema, info_keys)
   return schema
-
-def _get_bigquery_mode_from_vcf_num(vcf_num):
```

Comment:
Moving this function to another module seems to be independent of the scope of this change; is there any particular reason to do it in this very PR?

Reply:
Allie actually suggested it in this PR; it was a small enough change (and refactoring related), so I decided to do it in this PR. I can change it in another PR if you feel strongly about keeping the scope smaller.